src/main/java/nl/ordina/jobcrawler/service/SkillService.java

```diff
@@ -23,19 +23,40 @@ public SkillService(SkillRepository skillRepository) {
         this.skillRepository = skillRepository;
     }
 
-    //******** Getting ********//
-    public List<Skill> getAllSkills() {
+
+    @Override
+    public Optional<Skill> findById(UUID id) {
+        return skillRepository.findById(id);
+    }
+
+    @Override
+    public List<Skill> findAll() {
         return skillRepository.findAll();
     }
 
-    public Optional<Skill> getSkillByName(Skill skill) {
-        return skillRepository.findByName(skill.getName());
+    @Override
+    public boolean update(Skill skill) {
+        return false;   // TODO: implementing this function
```

Comment:
Dit lijkt me makkelijk en snel te implementeren. Neem dit nog aub mee in deze branch, voordat je em afrond.